### PR TITLE
Use new writeCharacteristic method for API Level >= 33

### PR DESCRIPTION
**Pull request by td-tomasz-joniec**

### CODE REVIEW

File: android/src/main/java/com/capacitorjs/community/plugins/bluetoothle/Device.kt [hunk 1]
```diff
@@ -388,9 +388,16 @@ class Device(
             return
         }
         val bytes = stringToBytes(value)
-        characteristic.value = bytes
-        characteristic.writeType = writeType
-        val result = bluetoothGatt?.writeCharacteristic(characteristic)
+
+        val result = if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.TIRAMISU) {
+            val statusCode = bluetoothGatt?.writeCharacteristic(characteristic, bytes, writeType)
+            statusCode == 0
```
Comment (Andy3189) [inline android/src/main/java/com/capacitorjs/community/plugins/bluetoothle/Device.kt:394]: this line doesnt look right

Comment (peitschie) [reply android/src/main/java/com/capacitorjs/community/plugins/bluetoothle/Device.kt:394]: I think this is just turning the int return into a bool. The `0` value here is https://developer.android.com/reference/android/bluetooth/BluetoothStatusCodes#SUCCESS

Comment (Andy3189) [reply android/src/main/java/com/capacitorjs/community/plugins/bluetoothle/Device.kt:394]: In that case i would suggest `bluetoothGatt?.writeCharacteristic(characteristic, bytes, writeType) == BluetoothStatusCodes.SUCCESS` for better readability

**Comment by pwespi** [comment]
Thank you for this PR and sorry for the late reply. 

I will accept it and refactor slightly before the next release.

**Comment by pwespi** [comment]
@all-contributors please add @td-tomasz-joniec  for code

**Pull request merged**